docs/release-notes/release-notes-0.14.0.md

```diff
@@ -0,0 +1,33 @@
+# Release Notes
+
+# RPC
+[The updatechanpolicy call now detects absent, pending and inactive channels and returns an 
+error](https://github.com/lightningnetwork/lnd/pull/5405).
+
+# Build System
+
+[A new pre-submit check has been
+added](https://github.com/lightningnetwork/lnd/pull/5520) to ensure that all
+PRs ([aside from merge
+commits](https://github.com/lightningnetwork/lnd/pull/5543)) add an entry in
+the release notes folder that at leasts links to PR being added.
+
+# Code Health
+
+## Code cleanup, refactor, typo fixes
+
+* [Unused error check 
+  removed](https://github.com/lightningnetwork/lnd/pull/5537).
+* [Shorten Pull Request check list by referring to the CI checks that are 
+  in place](https://github.com/lightningnetwork/lnd/pull/5545).
+* [Added minor fixes to contribution guidelines](https://github.com/lightningnetwork/lnd/pull/5503).
+
+## Database
+
+* [Ensure single writer for legacy
+  code](https://github.com/lightningnetwork/lnd/pull/5547) when using etcd
+  backend.
+
+# Contributors (Alphabetical Order)
+* ErikEk
+* Zero-1729
```

Comment:
First rebase your original commit onto latest master. Then add your changes to release notes